Projects/Chat/Sources/Navigation/ChatNavigation.swift

```diff
@@ -73,8 +84,17 @@ public struct ChatNavigation<Content: View>: View {
         RouterHost(router: router, options: .navigationType(type: .large), tracking: ChatNavigationViewName.chat) {
             Group {
                 switch chatType {
-                case let .conversationId(id):
-                    ChatScreen(vm: .init(chatService: ConversationService(conversationId: id)))
+                case let .conversationId(conversationId, claimId):
```

Comment:
Same, I don't think we need to pass claimId around.
ChatScreen model should know whenever there is a claim or not.